packages/atomizer/src/atomizer.js

```diff
@@ -532,6 +536,20 @@ Atomizer.prototype.getCss = function (config, options) {
                 }
 
                 const breakPoint = breakPoints && breakPoints[treeo.breakPoint];
+                if (treeo.breakPoint && !breakPoint) {
+                    const message = [
+                        `Class \`${treeo.className}\` contains breakpoint \`${treeo.breakPoint}\` which does not exist and must be manually added to your config file:`,
+                        '"breakPoints": {',
+                        `    "${treeo.breakPoint}": <YOUR-CUSTOM-VALUE>`,
+                        '}',
+                    ].join('\n');
+                    if (isStrict) {
+                        console.error('Error:', message);
+                        process.exit(1);
+                    } else if (isVerbose) {
+                        console.warn('Warning:', message);
```

Comment:
Note that this will still write the "bad" CSS. Without a valid breakpoint, `W(100%)--sm'` will yield

```css
.W\(100\%\)--sm {
  width: 100%;
}
```

Again the issue being that there is no media query so this is applied in all instances rather than

```css
@media(...) {
  .W\(100\%\)--sm {
    width: 100%;
  }
}
```